### PR TITLE
project setup

### PR DESCRIPTION
**Pull request by supritaj**

### CODE REVIEW

File: cabAd/settings.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
```
Comment (mohitsorde) [inline cabAd/settings.py:33]: Register the app

**Review by mohitsorde — CHANGES_REQUESTED** [review]
Now since we have folders remove the corresponding models.py, views.py, tests.py files ... Add a views folder as well

**Comment by mohitsorde** [comment]
Also add a middlewares module

**Comment by supritaj** [comment]
> Now since we have folders remove the corresponding models.py, views.py, tests.py files ... Add a views folder as well


> Now since we have folders remove the corresponding models.py, views.py, tests.py files ... Add a views folder as well

there is already a templates folder at root level, is views folder required then?

**Pull request closed without merging**